src/sage/math.yaml

```diff
@@ -28,9 +28,8 @@ descr: |
 
     Function calls are always `function_name(...)` with round brackets, too.
 code: |
-    # TODO: fix this expression
     var('x')
-    sqrt [ (4 + x) (2 - x) ]
+    sqrt( (4 + x) * (2 - x) ) # = sqrt [ (4 + x) (2 - x) ]
```

Comment:
ok, I see, I had no recollection of writing this, maybe it wasn't even me who wrote this originally. I think this "exercise" should be kept in, because it's targeting converted mathematica users. I'll come up with a scheme to tag such snippets, likely a field `test: false`.